### DISCUSSION TITLE
5.0文档bug，版本自动切换

### DISCUSSION THREAD

**Discussion by v2jun** [w7corp/easywechat#2599] [Q&A] [answered]
5.0的文档，当点入一个子类时（如公众号->入门）显示为文档正确内容，但是当再次点击该子类时，文档会自动切换到4.0。
查看文档过程中偶然发现内容和之前看的不一样，查看地址栏发现地址栏中的5.x再重复点击侧栏子类时会自动切换为4.x。

注：以切换浏览器测试，edge，Chrome都有同样问题，排除浏览器原因。

**Comment by overtrue** [comment, marked as answer]
https://github.com/vuejs/vitepress/pull/948

**Reply by overtrue** [reply]
已修复
